openmetadata-ui/src/main/resources/ui/src/components/PageLayoutV1/PageLayoutV1.tsx

```diff
@@ -99,27 +103,35 @@ const PageLayoutV1: FC<PageLayoutProp> = ({
             {leftPanel}
           </Col>
         )}
-        <Col
-          className={classNames(
-            'page-layout-v1-center p-t-sm page-layout-v1-vertical-scroll',
-            {
-              'flex justify-center': center,
-            },
-            mainContainerClassName
-          )}
-          flex={contentWidth}
-          offset={center ? 3 : 0}
-          span={center ? 18 : 24}>
-          {children}
-        </Col>
-        {rightPanel && (
+        <Col span={24}>
           <Col
-            className="page-layout-rightpanel page-layout-v1-vertical-scroll"
-            flex={rightPanelWidth + 'px'}
-            id="right-panelV1">
-            {rightPanel}
+            className={classNames(
+              'page-layout-v1-center p-t-sm page-layout-v1-vertical-scroll',
+              {
+                'flex justify-center': center,
+              },
+              mainContainerClassName
+            )}
+            flex={contentWidth}
+            offset={center ? 3 : 0}
+            span={center ? 18 : 24}>
+            <div className="alert-page-container">
+              {alert && <AlertBar message={alert.message} type={alert.type} />}
+              <div
+                className={`page-content ${alert && 'page-content-shifted'}`}>
```

Comment:
Do we need this div? also page-content-shifted?

Reply:
Yes this is needed. CSS is written as per the classes.